meson.build

```diff
@@ -245,6 +267,9 @@ features = {
   },
   'protobuf-support': {
     'extra_deps': [ pb_comp ]
+  },
+  'tensorrt-support': {
+    'extra_deps': tensorrt_dep
```

Comment:
It's ```extra_deps```, not ```extra_dep```; thus, ```[ tensorrt_dep ]``` will look more beautiful :)

Reply:
Ah, `tensorrt_dep` is actually 'deps'. I will rename it.

Reply:
That's right, updated.